Gruntfile.js

```diff
@@ -134,10 +134,10 @@ module.exports = function(grunt) {
 
   var browsers = [{
     browserName: "firefox",
-    version: "30"
+    platform: "linux"
   }, {
     browserName: "chrome",
-    version: "35"
+    platform: "linux"
```

Comment:
Why "linux"?  At the very least we probably want to go something like... Windows?